libs/barista-components/sunburst-chart/src/sunburst-chart.ts

```diff
@@ -135,27 +139,64 @@ export class DtSunburstChart {
     DtSunburstChartNode[]
   > = new EventEmitter();
 
-  /** Template portal for the default overlay */
-  // tslint:disable-next-line: use-default-type-parameter no-any
-  private _portal: TemplatePortal<any> | null;
+  /** @internal Overlay template */
+  @ContentChild(DtSunburstChartOverlay, { static: false, read: TemplateRef })
+  _overlay: TemplateRef<DtSunburstChartTooltipData>;
 
   /** Reference to the open overlay. */
-  private _overlayRef: OverlayRef;
+  private _overlayRef: DtOverlayRef<DtSunburstChartTooltipData> | null;
+
+  /** @internal Destroy subject to clear subscriptions on component destroy. */
+  private readonly _destroy$ = new Subject<void>();
 
   constructor(
-    private _overlayService: Overlay,
-    private _viewContainerRef: ViewContainerRef,
+    private _overlayService: DtOverlay,
     private _platform: Platform,
     // TODO: remove this sanitizer when ivy is no longer opt out
     private _sanitizer: DomSanitizer,
-    private _elementRef?: ElementRef<HTMLElement>,
-    @Optional()
-    @Inject(DT_UI_TEST_CONFIG)
-    private _config?: DtUiTestConfiguration,
+    private _elementRef: ElementRef<HTMLElement>,
+    private _resizer: DtViewportResizer,
+    private _cdr: ChangeDetectorRef,
```

Comment:
Could we do this without a breaking change as well? When making the new parameters optional and have a fallback to the way that it is currently working, we should be able to do this without a breaking change. 
As I assume that after we released version 7 a couple of weeks ago, version 8 will take some time to be released.

Reply:
Yeah please leave even unused parameters in there to avoid breaking changes. Version 8 is far out. No plans or timeline yet.

Reply:
@tomheller @ffriedl89 I can keep old params but for new params we're gonna loose the ability of scaling the chart on resize. I think it would be good to open a new PR for this so I'll leave the code commented